### PR TITLE
[example] Import generated artifacts.

### LINKED ISSUE #17: Check-in generated artifacts of example app?
It might be nice for people to be able to quickly see real world artefacts and also for us to update those along with plugin changes so that we can review this _besides_ any other tests. What do you think?

### PR DESCRIPTION
**Pull request by alloy**
Closes #17.

### CODE REVIEW

File: example/ts/__generated__/appQuery.graphql.ts [hunk 1]
```diff
@@ -0,0 +1,304 @@
+/* tslint:disable */
```
Comment (alloy) [inline example/ts/__generated__/appQuery.graphql.ts]: I actually want to create a TSLint rule that verifies that all properties of these fragment types are actually being used in user code to find any accidental over-fetching. This comment to disable TSLint completely is going to be interesting to deal with.

Comment (kastermester) [reply example/ts/__generated__/appQuery.graphql.ts]: The issue is that we can't emit TS code and be sure that we adhere to whatever tslint rules people use. It'd be great to have TSLint enabled in the emitted code, but how can we ever be sure that it is emitted in the style that the user wants?

I guess we could find all style rules and disable those?

Comment (alloy) [reply example/ts/__generated__/appQuery.graphql.ts]: I don’t think we need to attempt that, disabling the linter wholesale is imo the right decision (which is what relay-compiler does as well).

I was just kinda thinking out loud about how to deal with that, I’ll figure something out 👍

**Comment by kastermester** [comment]
> One thing that I noticed is that these artifacts use mixed indentation levels. The TS types use 4 spaces whereas the fragment JSON generated by Relay uses 2 spaces. Is this something we can control and change to match Relay’s behaviour?

I'm sure it is possible, but not quite sure how. I have used the following previously to turn string literals into single quotes. I would guess something similiar is needed to change indentation, but we might need some help from the TS team on this.

```ts
const printer = ts.createPrinter(
		{
			newLine: ts.NewLineKind.LineFeed,
		},
		{
			onEmitNode(
				hint: ts.EmitHint,
				node: ts.Node,
				emitCallback: (hint: ts.EmitHint, node: ts.Node) => void,
			): void {
				if (node != null) {
					if (node.kind === ts.SyntaxKind.StringLiteral) {
						(node as any).singleQuote = true;
					}
				}
				emitCallback(hint, node);
			},
		},
	);
```

**Comment by alloy** [comment]
I may have some time tonight to poke into the TS API.

**Pull request merged**